### PR TITLE
@types/sanitize-html Fix TypeScript version so #36587 can be unblocked.

### PR DESCRIPTION
**Pull request by paambaati**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/36587#issuecomment-507511778
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by paambaati** [comment]
`@types/sanitize-html` requires `@types/htmlparser2` but does not pin the TypeScript version. #36587 pins it to version 2.2 on `@types/htmlparser2`, so builds have been failing (see https://travis-ci.org/DefinitelyTyped/DefinitelyTyped/builds/553102250#L485).

File: types/sanitize-html/index.d.ts [hunk 1]
```diff
@@ -8,9 +8,13 @@
 //                 A penguin <https://github.com/sirMerr>
 //                 Johan Davidsson <https://github.com/johandavidson>
 //                 Jianrong Yu <https://github.com/YuJianrong>
+//                 GP <https://github.com/paambaati>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
```
Comment (WillGibson) [inline types/sanitize-html/index.d.ts]: I'm not convinced we should be locking this to TypeScript 2.2, e.g I know at least one place it's being used with TypeScript 3.x.

Comment (paambaati) [reply types/sanitize-html/index.d.ts]: @WillGibson This is the minimum version of TS that these typings are guaranteed to work with. It works fine on TS 3.x.

Comment (armanio123) [reply types/sanitize-html/index.d.ts]: @WillGibson as @paambaati mentioned is the minimum version. Here's a little [documentation](https://github.com/DefinitelyTyped/DefinitelyTyped/#i-want-to-use-features-from-typescript-21-or-above) about it.

**Comment by paambaati** [comment]
@rogierschouten @afshin @BehindTheMath @biermeester @WillGibson @sirMerr @johandavidson @YuJianrong **_BUMP!_**

**Comment by paambaati** [comment]
@rogierschouten @afshin @BehindTheMath @biermeester @WillGibson @sirMerr @johandavidson @YuJianrong **BUMP!**

File: types/sanitize-html/index.d.ts (hunk 1 above)
Comment (WillGibson) [reply types/sanitize-html/index.d.ts]: Thanks people.

**Review by WillGibson — APPROVED** [review]

**Pull request merged**